dev/forbidden_from_release_tests/README.md

```diff
@@ -0,0 +1,5 @@
+# Forbidden from release tests
+
+This program compiles the Dart portions of some Flutter applications to the
```

Comment:
What flutter app is actually used?

Reply:
By default it's using `examples/hello_world`. That's documented in the usage help from main.dart.

Reply:
Maybe add that information here as well? "some Flutter app" is so vague..

Reply:
Made some updates here - hopefully it's clearer now.